src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -6,19 +6,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import seedu.address.logic.commands.AddCommand;
-import seedu.address.logic.commands.ClearCommand;
-import seedu.address.logic.commands.Command;
-import seedu.address.logic.commands.DeleteCommand;
-import seedu.address.logic.commands.EditCommand;
-import seedu.address.logic.commands.ExitCommand;
-import seedu.address.logic.commands.FindCommand;
-import seedu.address.logic.commands.HelpCommand;
-import seedu.address.logic.commands.HistoryCommand;
-import seedu.address.logic.commands.ListCommand;
-import seedu.address.logic.commands.RedoCommand;
-import seedu.address.logic.commands.SelectCommand;
-import seedu.address.logic.commands.UndoCommand;
+import seedu.address.logic.commands.*;
```

Comment:
Try not to group every import together into using the kleene star.

Need to change a setting on your IntelliJ to prevent it from merging all together.

[Configure IntelliJ Coding Style](https://github.com/CS2103JAN2018-F14-B1/main/blob/master/docs/DeveloperGuide.adoc#241-configuring-the-coding-style)